### ISSUE TITLE
Implement training code

### ISSUE THREAD

**Issue by emu1200tra** [cydonia999/Tiny_Faces_in_Tensorflow#12]
Hi,

This is a very nice work. It helps my research a lot. However, when I try to implement training part based on your architecture, it counters with lots of problems. Therefore, will you implement training part and release in the future? I think it will help many people as well. Thx!

**Comment by MansiAgarwal11** [comment]
I completely agree. It's difficult to train on Matlab using remote servers and it'd be really helpful if you could share the training code too.

**Comment by pyupcgithub** [comment]
So, have someone know how to train using tensorflow ?
@emu1200tra @MansiAgarwal11 @cydonia999
